src/styles/navbar.module.scss

```diff
@@ -63,7 +69,7 @@
     position: absolute;
     height: 2px;
     width: 24px;
-    transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+    // transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
```

Comment:
Why was this commented out?

Reply:
While testing toggle button I commented this line,Even after commented out it is working fine.Now can i keep it or delete this line?